### ISSUE TITLE
Compilation error with PrettyPrinter

### ISSUE THREAD

**Issue by gkatev** [geany/geany-plugins#1059]
Hi,
I'm getting this error from the compilation of PrettyPrinter

```
  CC       pretty_printer_la-PluginEntry.lo
In file included from /usr/include/unicode/umachine.h:52,
                 from /usr/include/unicode/utypes.h:38,
                 from /usr/include/unicode/ucnv_err.h:88,
                 from /usr/include/unicode/ucnv.h:52,
                 from /usr/include/libxml2/libxml/encoding.h:31,
                 from /usr/include/libxml2/libxml/parser.h:810,
                 from PluginEntry.h:32,
                 from PluginEntry.c:28:
PrettyPrinter.h:55:22: error: two or more data types in declaration specifiers
   55 | typedef unsigned int bool;
      |                      ^~~~
In file included from PluginEntry.h:34,
                 from PluginEntry.c:28:
PrettyPrinter.h:55:1: warning: useless type name in empty declaration
   55 | typedef unsigned int bool;
      | ^~~~~~~
make[3]: *** [Makefile:664: pretty_printer_la-PluginEntry.lo] Error 1
```

I guess that this may be because the bool type is defined somewhere else?
Removing the typedef and including stdbool.h fixed it for me, but AFAIK it requires C99.

**Comment by elextr** [comment]
The plugin could always set its own requirements, but now Geany itself has moved to C99 (see HACKING) so the plugin should be upgraded too.

**Comment by codebrainz** [comment]
Duplicate of #1023 and fixed in #1053.
